src/Components/SoundWave/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useRef, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+
+const SoundWave = ({ audioFile, waveColor, progressColor }) => {
+  const waveFormRef = useRef(null);
+  const waveSurferRef = useRef(null);
+  const [isPlay, setIsPlay] = useState(false);
+
+  useEffect(() => {
+    waveSurferRef.current = window.WaveSurfer.create({
+      container: waveFormRef.current,
+      waveColor,
```

Comment:
Let's create default colors in case color props are not passed in.

- [ ] add default color values